src/condor_utils/file_transfer.cpp

```diff
@@ -3471,7 +3381,8 @@ FileTransfer::DoDownload( filesize_t *total_bytes_ptr, ReliSock *s)
 		for ( auto it = deferredTransfers.begin(); it != deferredTransfers.end(); ++ it ) {
 			int exit_status = 0;
 			FileTransferPlugin & plugin = Plugin(it->first);
-			TransferPluginResult result = InvokeMultipleFileTransferPlugin( errstack, exit_status, plugin, it->second, LocalProxyName.c_str(), false);
+			std::vector<ClassAd> resultAds; // in case we want to send these on
```

Comment:
This comment (and the parameter name) is a little deceptive; you only put ads in this vector if the `InvokeMultipleFileTransferPlugin()` fails to send them up the pipe.  It might be clearer to have `InvokeMultipleFileTransferPlugin()` always populate the vector and this code always try to send the ads up the pipe.